crates/astria-sequencer/src/app.rs

```diff
@@ -182,7 +182,7 @@ impl App {
     /// Generates a commitment to the `sequence::Actions` in the block's transactions
     /// and ensures it matches the commitment created by the proposer, which
     /// should be the first transaction in the block.
-    #[instrument(name = "App::process_proposal", skip(self))]
+    #[instrument(name = "App::process_proposal", skip(self, process_proposal))]
```

Comment:
Do we need to skip all of it or is there some info that's useful?

Also, if you want to skip all args, use `skip_all`

Reply:
i just skipped it all for ease - i guess the hash could be good to log